components/About.js

```diff
@@ -0,0 +1,47 @@
+import SectionTitle from './SectionTitle'
+export default function About () {
```

Comment:
Please change the component name to `AboutSection` and filename to `AboutSection.js` so that it is consistent with other section components.